.dockerignore

```diff
@@ -0,0 +1,2 @@
+dist/
+.git/
```

Comment:
DO include the `.git` folder as it is necessary in the top-level Makefile for generating version tags:
```diff
diff --git a/.dockerignore b/.dockerignore
index 43fe8fa..30b9507 100644
--- a/.dockerignore
+++ b/.dockerignore
@@ -1,2 +1,2 @@
 dist/
-.git/
\ No newline at end of file
+
```